forgettable-frontend/src/components/Dialog/Dialog.module.css

```diff
@@ -0,0 +1,55 @@
+.dialog {
+  position: fixed;
+  background: white;
+  width: 525px;
+  height: 400px;
+  z-index: 2;
+  border-radius: 4px;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  display: flex;
+  flex-direction: column;
+  /* justify-content: center;
+  justify-items: center;
+  align-content: center; */
+  gap: 20px;
+  color: #000;
+}
+
+.title{
+  font-size: 30px;
+  color: brown;
+  margin-top: 50px;
+  margin-left: 123px;
+  margin-right: 123px;
```

Comment:
`text-align: center` would be better here as we want to generally avoid hardcoding values